app/helpers/zendesk_service_helper.rb

```diff
@@ -98,6 +98,14 @@ def find_or_create_end_user(name, email, phone, exact_match: false, time_zone: n
     result.id if result.present?
   end
 
+  def qualified_environments
+    %w[development demo staging]
+  end
+
+  def qualify_user_name(name)
+    qualified_environments.include?(Rails.env) ? "#{name} (Fake User)" : name
+  end
+
```

Comment:
non-blocking but I wonder if there is a more specific name we could use for these methods. I wouldn't know what `qualify` means without having read the ticket. For example: `zendesk_testing_environments`, or `tagged_user_name` or `scoped_user_name` or even `zendesk_requester_name`?